src/shared/components/HeaderTextField/HeaderTextField.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import HeaderTextField, { Variant } from './HeaderTextField'
+
+export default {
+  title: 'Shared/HeaderTextField',
+  component: HeaderTextField,
+  argTypes: {
+    name: { control: 'text', defaultValue: 'Lorem ipsum' },
+    warningText: { control: 'text', defaultValue: 'Channel title must be at least 2 character' },
+    errorText: { control: 'text', defaultValue: 'Some error text' },
+    variant: {
+      control: {
+        type: 'select',
+        options: ['default', 'error', 'warning'],
+      },
+    },
+  },
+}
+
+type StoryProps = {
```

Comment:
Any reason to specify props explicitly? Doesn't the default controls approach work here?
```
const Template: Story<HeaderTextFieldProps> = (args) => (
```